src/app/tests/TestWriteInteraction.cpp

```diff
@@ -25,19 +28,50 @@
 #include <crypto/DefaultSessionKeystore.h>
 #include <lib/core/CHIPCore.h>
 #include <lib/core/ErrorStr.h>
+#include <lib/core/StringBuilderAdapters.h>
 #include <lib/core/TLV.h>
 #include <lib/core/TLVDebug.h>
 #include <lib/core/TLVUtilities.h>
 #include <lib/support/TestGroupData.h>
 #include <lib/support/TestPersistentStorageDelegate.h>
-#include <lib/support/UnitTestContext.h>
-#include <lib/support/UnitTestRegistration.h>
+#include <lib/support/tests/ExtraPwTestMacros.h>
 #include <messaging/ExchangeContext.h>
 #include <messaging/Flags.h>
+#include <pw_unit_test/framework.h>
 
-#include <memory>
-#include <nlunit-test.h>
-#include <utility>
+/**
+ * Helper macro we can use to pretend we got a reply from the server in cases
```

Comment:
It seems there is 3 definition of the same macro. Can you refactor it and make only one?